$OpenBSD$

Remove simplexmq source-repository-package from cabal.project.
simplexmq is provided as local source via ../simplexmq instead.

--- cabal.project.orig
+++ cabal.project
@@ -19,11 +19,6 @@
 constraints: zip +disable-bzip2 +disable-zstd

-source-repository-package
-    type: git
-    location: https://github.com/simplex-chat/simplexmq.git
-    tag: f03cec7a58ed13a39a52886888c74bcefdb64479
-
 source-repository-package
     type: git
     location: https://github.com/simplex-chat/hs-socks.git
